pkgs/development/python-modules/cpflows/default.nix

```
@@ -0,0 +1,78 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+, cycler
+, future
+, h5py
+, kiwisolver
+, matplotlib
+, numpy
+, pandas
+, pyparsing
+, python-dateutil
+, pytz
+, scikit-learn
+, scipy
+, seaborn
+, six
+, torch
+, torchvision
+, tqdm
+, unittestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "cpflows";
+  version = "0.1.0";
+  format = "setuptools";
+
+  disable = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "lostella";
+    repo = "CP-Flow";
+    rev = "96307119fe734c3226b0f685ddaf10f0004fb177"; # gluonts-dependency branch head
+    hash = "sha256-VtDDjpOv85GbBqv0wyyw4A4XOV+8aoLz8XGM0IB2Fec=";
+  };
+
+  patches = [
+    ./fix-dependencies.patch
+  ];
+
```

```suggestion
  patches = [
    ./fix-dependencies.patch
  ];
```

Could you please check if it's possible to use `pythonRelaxDepsHook` here? Editing/creating patch files for updates is always a bit of a hassle from my point of view. If the hook doesn't work a `postPatch` section with `substituteInPlace` could also do the job.

---

Fixed